src/e50hx/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set (libname "e50hx")
+set (libdescription "upm module for the Veris E50HX (E50H2/E50H5)Energy Meters")
+set (module_src ${libname}.cxx)
+set (module_h ${libname}.h)
+
+pkg_search_module(BACNET libbacnet)
+if (BACNET_FOUND)
+  # upm-libbacnetmstp will bring in libbacnet, I hope
+  set (reqlibname "upm-bacnetmstp")
+  include_directories(${BACNET_INCLUDE_DIRS})
+  include_directories("../bacnetmstp")
+  upm_module_init()
+  add_dependencies(${libname} bacnetmstp)
```

Comment:
Yes, I've been reading through the MRAA and UPM logs, PR's and issues this morning to catch up.  Do you want me to fix this?